playbooks/README.md

```diff
@@ -159,7 +159,7 @@ No additional cni plugin will be deployed.
 ### SELinux
 
 In case you are experiencing permission or SELinux issues, please consider
-creating an issue for [kubevirt](https://github.com/kubevirt/kubevirt/issues/) 
+creating an issue for [kubevirt](https://github.com/kubevirt/kubevirt/issues/)
```

Comment:
Not sure if it need the space here, I think no.

Reply:
Exactly, we don't need any space here.

Reply:
:( sorry, I mean it need a space here. Just forget type "not".

Reply:
I think it does not require any space `an issue for kubevirt and report what is `
I have checked in my forked repo.

Reply:
you may have a look here https://github.com/adityaramteke/kubevirt-ansible/blob/invalide_url/playbooks/README.md#selinux